meta-genivi-dev/recipes-dev-platform/packagegroups/packagegroup-gdp-sdl.bb

```diff
@@ -0,0 +1,18 @@
+DESCRIPTION = "Smart Device Link (SDL) package group"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${IVI_COREBASE}/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+# Avoid hardcoding the full layer path into the checksums
```

Comment:
Is this a FIXME comment? If so, please append "FIXME: " so it's clear that it's what you should be doing rather than what you are already doing.